openmc/cmfd.py

```diff
@@ -196,13 +196,10 @@ class CMFDRun(object):
     ----------
     tally_begin : int
         Batch number at which CMFD tallies should begin accummulating
-    feedback_begin: int
-        Batch number at which CMFD feedback should be turned on
+    cmfd_begin: int
```

Comment:
How about `solver_begin`? Given that the class is `CMFDRun`, to me `cmfd_` is somewhat redundant.